src/main/java/com/ibm/etcd/client/EtcdClient.java

```diff
@@ -665,11 +670,10 @@ public void applyRequestMetadata(RequestInfo requestInfo,
                             failStatus = Status.UNAUTHENTICATED
                                     .withDescription("(Re)authentication RPC failed")
                                     .withCause(failure);
-                            authFailRetryTime = System.currentTimeMillis() + 5_000L;
+                            authFailRetryTime = System.currentTimeMillis() + AUTH_RETRY_TIMEOUT_MS;
                         } else {
                             failStatus = Status.fromThrowable(failure);
-                            // If this was a real auth failure, postpone further attempts a bit longer
-                            authFailRetryTime = System.currentTimeMillis() + 15_000L;
+                            authFailRetryTime = System.currentTimeMillis() + AUTH_RETRY_TIMEOUT_MS;
```

Comment:
Waiting for 15s every time I hot reload is bad. Most likely it is good to turn the delay off (or make it a parameter).